PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackV0Extended.cxx

```diff
@@ -459,6 +510,129 @@ struct femtoUniversePairTaskTrackV0Extended {
   }
 
   PROCESS_SWITCH(femtoUniversePairTaskTrackV0Extended, processMixedEventV0, "Enable processing mixed events for V0 - V0", false);
+
+  ///--------------------------------------------MC-------------------------------------------------///
+
+  using FemtoRecoParticles = soa::Join<aod::FDParticles, aod::FDExtParticles, aod::FDMCLabels>;
+
+  /// This function fills MC truth particles from derived MC table
+  void processMCTruth(aod::FDParticles const& parts)
+  {
+    for (auto& part : parts) {
+      if (!(part.partType() == uint8_t(aod::femtouniverseparticle::ParticleType::kMCTruthTrack)))
+        continue;
+      if (TMath::Abs(part.eta()) > ConfEta || part.pt() < ConfLPtMC || part.pt() > ConfHPtMC)
+        continue;
+
+      int pdgCode = static_cast<int>(part.pidcut());
+      const auto& pdgParticle = pdgMC->GetParticle(pdgCode);
```

Comment:
Do you really need pdgParticle here? Wouldn't a check on the sign of pidcut be enough?

Reply:
Thanks for the comment. In some cases, the pdgCode does not imply charge of the particle (for example, anti-lambdas), that's why I am using it this way.